selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java

```diff
@@ -72,10 +72,10 @@
 
   @BeforeClass
   public void prepare() throws Exception {
-    URL resource = getClass().getResource("/projects/git-pull-test");
     testUserPreferencesServiceClient.addGitCommitter(gitHubUsername, productUser.getEmail());
+    URL resource = getClass().getResource("/projects/git-pull-test");
```

Comment:
Random change or does this fix something?

Reply:
Random change.